src/javascript/ContentEditor.constants.js

```diff
@@ -68,7 +68,8 @@ export const Constants = {
     },
     automaticOrdering: {
         section: 'listOrdering',
-        mixin: 'jmix:orderedList'
+        mixin: 'jmix:orderedList',
+        mixinAutomaticallyOrderable: 'jmix:automaticallyOrderable'
```

Comment:
We should continue to test on jmix:orderedList and not jmix:automaticallyOrderable,
so we could remove this jmix:automaticallyOrderable constant